lib/src/components/NxToggle/__tests__/NxToggle.test.tsx

```diff
@@ -5,116 +5,97 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import '@testing-library/jest-dom';
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import NxToggle, { Props } from '../NxToggle';
 
 describe('NxToggle', function() {
   const simpleProps: Props = {
-        inputId: 'toggle-id',
-        isChecked: false,
-        onChange: () => {},
-        disabled: undefined,
-        children: 'Enables whales'
-      },
-      getShallow = getShallowComponent<Props>(NxToggle, simpleProps);
+    inputId: 'toggle-id',
+    isChecked: false,
+    onChange: () => {},
+    disabled: undefined,
+    children: 'Enables whales'
+  };
 
-  it('renders a <label> containing a toggle <input> and toggle control', function() {
-    const shallowRender = getShallow();
+  it('renders a <label> with an <input>', function() {
+    render(<NxToggle {...simpleProps}></NxToggle>);
```

Comment:
It seems to me like we'd still benefit from a wrapper similar to `getShallowComponent` that encapsulates the `render()` call, the name of the component being rendered, and the minimal props.

Reply:
> It seems to me like we'd still benefit from a wrapper similar to `getShallowComponent` that encapsulates the `render()` call, the name of the component being rendered, and the minimal props.

Added render wrapper here: 95a4049c1f778e0213ca03d4d8ce4985a6995b00